src/plugins/wizard/public/application/utils/wizard_state_validation.test.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { validateWizardState } from './wizard_state_validation';
+
+describe('wizard state validation', () => {
```

Comment:
nit: We dont need to be so explicit in the test. We can assume that the ajv library performs as expected and simply test the logic that we have introduced. i.e. Lines 14 to 16 for `src/plugins/wizard/public/application/utils/wizard_state_validation.ts`

That will make this test more readable and more useful long term.

Reply:
Make sense. i reduce the unit test to only testing the logics that we introduce.